app/upgrades/v1/mainnet/finality_params.go

```diff
@@ -1,13 +1,11 @@
 package mainnet
 
-// TODO Some default parameters. Consider how to switch those depending on network:
-// mainnet, testnet, devnet etc.
 const FinalityParamStr = `{
-  "max_active_finality_providers": 100,
-  "signed_blocks_window": 100,
+  "max_active_finality_providers": 60,
+  "signed_blocks_window": 10000,
   "finality_sig_timeout": 3,
-  "min_signed_per_window": "0.1",
-  "min_pub_rand": 100,
-  "jail_duration": "86400s",
-  "finality_activation_height": 17500
+  "min_signed_per_window": "0.3",
+  "min_pub_rand": 8192,
```

Comment:
@gitferry does this need to be revisited with the new `w`?

Reply:
I don't think the value of `w` matters because it affects the timestamping delay. To compensate a large `w`, fp can be adapted to prepare multiple commits. Besides, Cubist's limit is 10k, right? Would it make sense to change it to 10k? Seems 8192 is difficult to remember 😅

Reply:
Right now the fp is not adapted to do that though right?

Reply:
>Seems 8192 is difficult to remember 
is is 2 to power 13, so it is not that bad.

Eitherway lets give Cubist a bit of a breathing room i.e they said their limits is 10k, so they should be fine with something below that 
>Right now the fp is not adapted to do that though right?
yup, but problem is entirely on fp config side